frontend/src/audio.html

```diff
@@ -0,0 +1,271 @@
+---
+pagination:
+  data: audios
+  size: 1
+  alias: audio
+permalink: "/audios/{{ audio.objectID | slug }}/"
+layout: base
+jsFile: audio.js
+---
+<article
+  class="m-10 flex flex-col"
+  id="audio-page"
+  data-audio-number="{{ audio.objectID }}"
+  data-audio-title="{{ audio.title }}"
+  data-dateforhumans="{{ audio.dateForHumans }}"
+  data-dateuncertain="{{ audio.dateUncertain }}"
+  data-location="{{ audio.location }}"
+  data-locationuncertain="{{ audio.locationUncertain }}"
+  data-category="{{ audio.category }}"
+  data-languages="{{ audio.languages | json | escape }}"
+  data-duration="{{ audio.duration }}"
+>
+  <div class="flex flex-col">
+    <div class="flex justify-between align-baseline">
+      <!-- Title -->
+      <span class="text-lg font-bold">{{ audio.title }}</span>
+      <!-- Duration -->
+      <span>{{ audio.duration | duration }}</span>
+    </div>
+    <div class="mt-1 flex flex-col justify-between md:flex-row">
+      {% render 'partials/tagline.html', audio-id: audio.objectID, audio-date: audio.dateForHumans, audio-location: audio.location, audio-category: audio.category, audio-languages: audio.languages -%}
+      <div class="flex justify-end pt-1 md:pt-0">
+        <div class="flex">
+          <div class="flex self-stretch">
+            <span class="p-1">
+              <div
+                class="{{ audio.soundQualityRating | soundquality }} flex flex-none space-x-1 rounded-lg border px-1.5 text-xs"
+                title="Sound quality of the recording"
+              >
+                <svg
+                  class="h-4 w-4 flex-none stroke-current stroke-2"
+                  xmlns="http://www.w3.org/2000/svg"
+                  viewBox="0 0 512 512"
+                >
+                  <path
+                    d="M83 384c-13-33-35-93.37-35-128C48 141.12 149.33 48 256 48s208 93.12 208 208c0 34.63-23 97-35 128"
+                    fill="none"
+                    stroke="currentColor"
+                    stroke-linecap="round"
+                    stroke-linejoin="round"
+                    stroke-width="32"
+                  />
+                  <path
+                    d="M108.39 270.13l-13.69 8h0c-30.23 17.7-31.7 72.41-3.38 122.2s75.87 75.81 106.1 58.12h0l13.69-8a16.16 16.16 0 005.78-21.87L130 276a15.74 15.74 0 00-21.61-5.87zM403.61 270.13l13.69 8h0c30.23 17.69 31.74 72.4 3.38 122.19s-75.87 75.81-106.1 58.12h0l-13.69-8a16.16 16.16 0 01-5.78-21.87L382 276a15.74 15.74 0 0121.61-5.87z"
+                    fill="none"
+                    stroke="currentColor"
+                    stroke-miterlimit="10"
+                    stroke-width="32"
+                  />
+                </svg>
+                <span>{{ audio.soundQualityRating }}</span>
```

Comment:
Here we should map the value to the label as per `soundQualityRatingMapping`. Compare to the [client-side code](https://github.com/PureBhaktiArchive/archive-web-app/blob/c9ecf78c335fa7231e9f0adbcaf817602017bc41/frontend/src/audio-item-template.js#L211).